views/deductions/trillium-rent-amount.pug

```diff
@@ -1,12 +1,12 @@
 extends ../base
 
 block variables
-  -var title = __('Enter rent paid in 2018')
+  -var title = 'Enter rent paid in 2018'
```

Comment:
Not concerned about it right now, but going forward we should stick to a consistent way for this

Reply:
Good point, will fix (and will update the "review" page as well)